test/notifications.test.mo

```diff
@@ -125,33 +121,30 @@ do {
   assert state() == (15, 0, 1); // state from before
   ignore await* handler.fetchFee();
   assert journal.hasEvents([
-    #issued(-5),
     #feeUpdated({ new = 15; old = 10 }),
   ]);
-  assert state() == (0, 0, 0); // state changed
-  assert (await f2) == ?(20, 5); // credit = latest - new_fee
+  assert state() == (15, 0, 1); // state not changed
+  assert (await f2) == ?(5, 5); // credit = latest - new_fee
```

Comment:
The comment in this line is wrong. Credit here is latest balance (20) minus old balance (15).